docker-compose.yml

```diff
@@ -1,5 +1,7 @@
 hawkular:
-  image: "hawkular/hawkular-services:latest"
+  # !! WARNING !! FIXME !! TODO !!
```

Comment:
So, what's the rightest docker tag now? Is it "hawkular-services:latest", for master?